app/src/main/java/ro/code4/deurgenta/ui/settings/SettingsFragment.kt

```diff
@@ -47,18 +37,30 @@ class SettingsFragment : Fragment() {
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
-        report_problem.setOnClickListener {
+        report_problem.setOnClickListener {reportProblemClicked(it)}
+    /* report_problem.setOnClickListener {
             Intent(Intent.ACTION_SEND).apply {
                 // The intent does not have a URI, so declare the "text/plain" MIME type
                 type = "text/html"
                 putExtra(Intent.EXTRA_EMAIL, arrayOf("jan@example.com")) // recipients
                 putExtra(Intent.EXTRA_SUBJECT, "Email subject")
                 putExtra(Intent.EXTRA_TEXT, "Email message text")
-                putExtra(Intent.EXTRA_STREAM, Uri.parse("content://path/to/email/attachment"))
-                // You can also attach multiple items by passing an ArrayList of Uris
             }
-        }
+        }*/
     }
 
+    private fun reportProblemClicked(view: View?) {
+        val emailIntent = Intent(
+                Intent.ACTION_SENDTO,
+                contactEmailUri
+        )
+        startActivity(
+                Intent.createChooser(
+                        emailIntent,
+                        "Send email via"
```

Comment:
this could be extracted in the strings.xml resource files so it will easily be translated :)